roles/ceph-mon/tasks/docker/fetch_configs.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: try to fetch ceph config and keys
   copy:
-    src: "{{ playbook_dir }}/{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
+    src: "{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
```

Comment:
The issue arises when ``fetch_directory`` is set to an absolute path, which is what CI does (and the ceph-installer project). Including ``playbook_dir`` here removes the users ability to set ``fetch_directory`` to anything other than a path relative to the playbook.

I didn't test this from a playbook from within ``infrastructure-playbooks/``, but isn't our current guidance there to move those playbooks up to the project root anyway?